src/cryptonote_basic/txtypes.h

```diff
@@ -28,12 +28,13 @@ enum class txtype : uint16_t {
     ethereum_service_node_exit_request,
     ethereum_service_node_exit,
     ethereum_staking_requirement_updated,
+    ethereum_new_service_node_v2,
```

Comment:
I'd like to try to avoid needing to use new tx types for different versions of the same conceptual tx type.  See my other, longer comment about how we might do that.

Reply:
Similar as above, it's temporary and to be dropped by launch. If we were to get rid of it now I guess is what's the way to retroactively figure out if the NewSN payload has a version or not (maybe a magic value in the header?)